dts/bindings/hwinfo/litex,dna0.yaml

```diff
@@ -1,20 +1,13 @@
-#
 # Copyright (c)  2019 Antmicro <www.antmicro.com>
-#
 # SPDX-License-Identifier: Apache-2.0
-#
 
 title: LiteX DNA
-description: >
-    This binding gives a base representation of the LiteX DNA
-inherits:
-        !include base.yaml
+description: LiteX DNA ID reader
 
-properties:
-    compatible:
-      constraint: "litex,dna0"
+compatible: "litex,dna0"
+
+include: base.yaml
 
+properties:
     reg:
       category: required
```

Comment:
Did you miss changing `category` to `required: true`

Reply:
Ohh... good catch.

Reply:
Fixed